components/pages/explorer/Jbrowse/JbrowseSelectedFilesTable.tsx

```diff
@@ -141,20 +149,30 @@ const JbrowseSelectedFilesTable = () => {
         margin-top: 20px;
       `}
     >
-      {hasWarnings && (
+      {!!compatibilityWarnings.length && (
```

Comment:
This may be a tad more declarative than Boolean casting
```suggestion
      {compatibilityWarnings.length > 0 && (
```